dfm_tools/meshkernel_helpers.py

```diff
@@ -77,6 +77,43 @@ def meshkernel_delete_withgdf(mk:meshkernel.meshkernel.MeshKernel, coastlines_gd
                          delete_option=meshkernel.DeleteMeshOption(2), #ALL_COMPLETE_FACES/2: Delete all faces of which the complete face is inside the polygon
                          invert_deletion=False)
 
+def meshkernel_delete_withshp(mk:meshkernel.meshkernel.MeshKernel, coastlines_shp, min_area: float=0):
+    """
+    Delete parts of mesh that are inside the shapefile polygon.
+
+    Parameters
+    ----------
+    mk : meshkernel.meshkernel.MeshKernel
+        DESCRIPTION.
+    coastlines_shp : path to the shp file
+        DESCRIPTION.
+
+    Returns
+    -------
+    None.
+
+    ---------------
+    Contributed by Roger Wang (rq.wang@rutgers.edu)
+    
+    """
+    if coastlines_shp == None:
+        print("Need a shape file.")
```

Comment:
This is not a possible situation, since the coastlines_shp argument is not optional. So this can be removed.